### PR TITLE
WIP: ENH: Update for ITK 5.2.0.post2

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by aylward** [comment]
Please rebase.  Just committed changes to github workflow/actions to include a static build of VTK and SlicerExecutionModel during dashboard / wheels so that applications can be built.

**Comment by thewtex** [comment]
@aylward 👍 ! rebased

**Comment by aylward** [comment]
LGTM!!!

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -274,6 +274,7 @@ jobs:
       matrix:
         include:
           - itk-python-git-tag: "v5.2.0"
```
Comment (aylward) [inline .github/workflows/build-test-package.yml]: This line needs to be removed.

Comment (thewtex) [reply .github/workflows/build-test-package.yml]: Good catch. Fixed.

**Pull request merged**